docs/generic-skills.md

```diff
@@ -0,0 +1,275 @@
+The AEA generic skills demonstrate an interaction between two AEAs.
+
+* The provider of the data.
+* The buyer of the data.
+
+## Preparation instructions
+ 
+### Dependencies
+
+Follow the <a href="../quickstart/#preliminaries">Preliminaries</a> and <a href="../quickstart/#installation">Installation</a> sections from the AEA quick start.
+
+##Discussion
+
+The scope of the specific demo is to demonstrate how to create an easy configurable AEA. The seller AEA
+will read data from `skill.yaml` file and will deliver to the buyer upon payment. To keep the demo simple we avoided the usage of a database or the usage of a hardware sensor since this would increase the complexity.
```

Comment:
will sell the service specified in the `skill.yaml` file and deliver it upon payment by the buyer.

Reply:
You can add, that adding a database or hardware sensor for loading the data is trivial.